openvdb_houdini/openvdb_houdini/CMakeLists.txt

```diff
@@ -207,7 +207,7 @@ if(OPENVDB_ENABLE_RPATH)
     ${Tbb_LIBRARY_DIRS}
   )
   if(OPENVDB_BUILD_CORE)
-    list(APPEND "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+    list(APPEND RPATHS ${CMAKE_INSTALL_FULL_LIBDIR})
   else()
     list(APPEND ${OpenVDB_LIBRARY_DIRS})
```

Comment:
@Idclip I guess that the `RPATHS` is missing here, am I right?

Reply:
Yeah another good spot - our CI builds with RPATHS enabled so I'm surprised neither of these commands cause a CMake error :|

Reply:
Fixed on 1f506b8709!